hack/make-rules/verify.sh

```diff
@@ -48,6 +48,7 @@ QUICK_PATTERNS+=(
   "verify-staging-client-go.sh"
   "verify-test-images.sh"
   "verify-test-owners.sh"
+  "verify-spelling.sh"
```

Comment:
nit: this list is alphabetized - so please move this up before "verify-staging-client-go.sh".

Reply:
also, I'm not sure whether 1m counts as "quick".